docs/scripting-ref.rst

```diff
@@ -331,6 +331,32 @@ Compare:
 See also: :ref:`splash-runjs`, :ref:`splash-jsfunc`,
 :ref:`splash-wait-for-resume`, :ref:`splash-autoload`.
 
+.. _splash-disablejs:
+
+splash:disablejs
```

Comment:
@immerrr Yeah, I agree that having separate `disable` and  `enable` functions is a bit redundant. Should I update the pr or wait for @kmike's response?